app/services/create_account_subscription_service.rb

```diff
@@ -26,6 +27,7 @@ def call
       subscriber_list_id: subscriber_list.fetch("id"),
       address: subscriber.fetch("address"),
       frequency: frequency,
+      skip_confirmation_email: @skip_confirmation_email,
```

Comment:
Actually, I think this should have always had `skip_confirmation_email: true` - whoops.  So this doesn't need a separate `@skip_confirmation_email` parameter here.

Reply:
Hmm yes, that makes sense. I'll drop the commit where I add it, sort this PR out then I'll open a new one that sets `skip_confirmation_email: true` in all cases and update the tests.